src/router/index.js

```diff
@@ -1,14 +1,13 @@
 import { createRouter, createWebHistory } from 'vue-router'
 import DataSessions from '../views/DataSessionsView.vue'
 import ProjectView from '../views/ProjectView.vue'
-import Home from '../views/HomeView.vue'
 import Registration from '../views/RegistrationView.vue'
 
 const routes = [
 	{
 		path: '/',
-		name: 'Home',
-		component: Home
+		name: 'Registration',
+		component: Registration
```

Comment:
If we navigate to the root / page and we're already logged in (e.g. auth data exists in store) does this redirect to the appropriate place, e.g. projects page?

Reply:
I didn't implement either of these since I wasn't sure if we'd be combining this with photon ranch in the future. But that would be so far out now that I think about it and the logic for rerouting shouldn't take too long. I'll add that now.